ops/jenkins/bfd-build-apps/Jenkinsfile

```diff
@@ -103,6 +103,9 @@ spec:
 """
         }
     }
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '15', daysToKeepStr: '30'))
+    }
 
```

Comment:
Newbie question: How come you don't set this in the jenkins files for bfd-availability-checker and bfd-availability-checker-build?

Reply:
Good FInd! I'm leaving bfd-availability-checker/Jenkinsfile alone since that job already has a configuration set. Updated bfd-availability-checker-build